src/nginxconfig/i18n/en/templates/domain_sections/index.js

```diff
@@ -29,10 +29,11 @@ import logging from './logging';
 import php from './php';
 import presets from './presets';
 import python from './python';
+import rails from './rails';
```

Comment:
Ditto -- Can you bump the copyright year in this file as we're touching it, please?